docs/data_sources/wip_future.rst

```diff
@@ -6,37 +6,36 @@ Work in Progress & Future Datasets
 
 .. contents::
 
-
 -------------------------------------------------------------------------------
 Work in Progress
 -------------------------------------------------------------------------------
 
-Thanks to a grant from the `Alfred P. Sloan Foundation Energy & Environment
-Program <https://sloan.org/programs/research/energy-and-environment>`__, we
-have support to integrate the following new datasets.
+Thanks to a grant from the `Alfred P. Sloan Foundation Energy & Environment Program
+<https://sloan.org/programs/research/energy-and-environment>`__, we have support to
+integrate the following new datasets.
 
 
 .. _data-tmolmp:
 
 ISO/RTO LMP
 ^^^^^^^^^^^
 
-Locational marginal electricity pricing information from the various grid
-operators (e.g. MISO, CAISO, NEISO, PJM, ERCOT...). At high time resolution,
-with many different delivery nodes, this will be a very large dataset (hundreds
-of GB). The format for the data is different for each of the ISOs. Physical
-location of the delivery nodes is not always publicly available.
+Locational marginal electricity pricing information from the various grid operators
```

Comment:
oh we are not planning on integrating this... maybe we should remove it?

Reply:
swapped it with 861